docs/docs/building-apps/03-app-upgrade.md

```diff
@@ -52,6 +52,12 @@ be a matter of minutes and not even require them to be awake at that time.
 
 ## Integrating With An App
 
+After app initiation, call `SetMigrationModuleManager` with ModuleManager to give BaseApp access to `RunMigrationBeginBlock`:
```

Comment:
We need to add a `:::tip` (https://docusaurus.io/docs/next/markdown-features/admonitions) that says the same thing about not being necessary for users of app v2 / depinject.